CMakeLists.txt

```diff
@@ -42,8 +42,8 @@ configure_file(CTestCustom.cmake CTestCustom.cmake)
 
 unset(AKTUALIZR_CHECKED_SRCS CACHE)
 
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
That isn't the default?

Reply:
This property is True by default for SHARED and MODULE library targets and False otherwise.